dev/RadioMenuFlyoutItem/RadioMenuFlyoutItem_rs2_themeresources.xaml

```diff
@@ -2,103 +2,89 @@
 <ResourceDictionary
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:contract7Present="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract,7)"
     xmlns:local="using:Microsoft.UI.Xaml.Controls">
 
     <Style TargetType="local:RadioMenuFlyoutItem" BasedOn="{StaticResource DefaultRadioMenuFlyoutItemStyle}" />
 
-    <!-- Same as default rs2 style for ToggleMenuFlyoutItem -->
+    <!-- Same as DefaultToggleMenuFlyoutItemStyle -->
     <Style x:Key="DefaultRadioMenuFlyoutItemStyle" TargetType="local:RadioMenuFlyoutItem">
```

Comment:
curious, what is different between the different versions ? Are all these the same now ? If so, could we just reuse the same file ?

Reply:
they're all the same now as ToggleMenuFlyoutItem is the same for all the versions:)